src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -183,6 +183,9 @@
       <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.DocumentDB.1.0.1-alpha1-10427\lib\net45\Microsoft.Azure.WebJobs.Extensions.DocumentDB.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Azure.WebJobs.Extensions.Http">
+      <HintPath>..\..\..\azure-webjobs-sdk-extensions\src\WebJobs.Extensions.Http\bin\Debug\Microsoft.Azure.WebJobs.Extensions.Http.dll</HintPath>
```

Comment:
Since I haven't published the package yet, I'm referencing the locally built assembly. Once the other PR is in and the package is published, I'll pull it in. I do this so I can run all tests - they're passing.